package.json

```diff
@@ -32,8 +32,9 @@
 	"dependencies": {
 		"@asyncapi/generator-filters": "^2.1.0",
 		"@asyncapi/generator-react-sdk": "^0.2.11",
-		"@asyncapi/modelina": "^0.59.1",
-		"@asyncapi/parser": "^1.15.1",
+		"@asyncapi/modelina": "^1.0.0-next.24",
+		"@asyncapi/parserV2": "npm:@asyncapi/parser@v2.0.0-next-major.10",
```

Comment:
If I dont include this dependency... It fails... 🤔 ??

Reply:
Where do you use old `@asyncapi/modelina`? 😅  Also, you have a lot of conflicts.

Reply:
Should be fixed @magicmatatjahu, I am not using the old one 🙂

Reply:
So why you cannot remove it from deps? Sorry I don't understand 😄

Reply:
So for the parser, the dependency have to be there `		"@asyncapi/parserV2": "npm:@asyncapi/parser@v2.0.0-next-major.10",`, Modelina dependency is of course updated 🙂

Reply:
aaa ok. `@asyncapi/parser` is still in generator 🤦🏼‍♂️ Sorry!

Reply:
We should remember about moving to the `@asyncapi/parser` in deps when we will merge generator PR.

Reply:
Yep 👍